src/compiler/diagnosticMessages.json

```diff
@@ -1408,6 +1408,22 @@
         "category": "Error",
         "code": 1443
     },
+    "'{0}' is a type and must be imported with a type-only import when 'preserveValueImports' and 'isolatedModules' are both enabled.": {

```

Comment:
This is a minor nit, but "... imported *with a* type-only import ..." sounds somewhat confusing and circular. Would any of the following work?

- "... imported *as a* type-only import ..."
- "... imported *using a* type-only import ..."

The same ambiguity applies to two messages following this one as well.

Reply:
Switching to “using”